okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java

```diff
@@ -16,26 +16,58 @@
 
 package com.palantir.remoting3.okhttp.metrics;
 
+import static com.google.common.base.Preconditions.checkState;
+
 import com.codahale.metrics.Meter;
 import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.ImmutableMap;
+import com.palantir.tritium.tags.TaggedMetric;
+import java.util.Map;
+import java.util.Optional;
+import java.util.SortedMap;
 
 public final class HostMetrics {
 
+    private static final String SERVICE_NAME_KEY = "service-name";
+    private static final String HOSTNAME_KEY = "hostname";
+    private static final String FAMILY_KEY = "family";
```

Comment:
should we expose these as either public constants or via static method (to allow changes so the strings aren't embedded in dependencies)?

Reply:
exposed publicly now